### ISSUE TITLE
Cannot override final method Brick\Math\Internal\Calculator::modInverse()

### ISSUE THREAD

**Issue by csoutham** [pinkary-project/pinkary.com#592]
With a fresh clone of the repo, running PHP v8.3.11 via Valet, upon visiting /register I run into a fatal error:
`Cannot override final method Brick\Math\Internal\Calculator::modInverse()`

![Screenshot 2024-09-09 at 19 45 17](https://github.com/user-attachments/assets/37dc0d73-0a82-4586-8f55-e31334ce4505)

I've not experienced this with any other Laravel v11/Livewire project.
Is it a PHP configuration issue, or am I missing an extension?

**Comment by mademan12** [comment]
This is a strange error, the [math](https://github.com/brick/math) library can work on any PHP installation but anyway, try to install one of these two extensions [GMP](http://php.net/manual/en/book.gmp.php) or [BCMath](http://php.net/manual/en/book.bc.php), if that doesn't work, try the traditional way to fix problems, remove the vendor folder and then run the install command  `composer install`.
I hope that helps you

**Comment by perisicnikola37** [comment]
@csoutham Any update on this one?

**Comment by csoutham** [comment]
@perisicnikola37 @mademan12 Yes, I've just tested it and removing the vendor folder (even though it was a clean clone) has fixed it. Bizarre, but thankful that's the fix!
